sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -31,7 +31,7 @@ class WordCountIT(unittest.TestCase):
   @attr('IT')
   def test_wordcount_it(self):
     # Set extra options to the pipeline for test purpose
-    extra_opts = {'on_success_matcher': PipelineStateMatcher()}
```

Comment:
Any reason not to use the all or any of pattern for matchers?

This allows the caller to choose whether all conditions need to be met, or some conditions or some combination of both.